src/pages/contact-us.js

```diff
@@ -153,8 +162,8 @@ const ContactUs = () => (
       .social-media-labels {
         display: flex;
         justify-content: space-between;
-        margin-right: 10vh;
-        width: 15vh;
+        margin-right: 6.5vw;
+        width: 9vw;
```

Comment:
This may be easier to use a fixed size, not a viewport unit. 6.5vw can be a lot on a large screen!

![image](https://user-images.githubusercontent.com/32434620/203790750-fb26d411-e321-4d0b-bc2e-b8828cf129ad.png)